plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -201,27 +206,26 @@ static int allocate_process_entry(CriuKfd *e)
 	return 0;
 }
 
-static int allocate_devinfo_entries(CriuKfd *e, int num_of_devices)
+static int allocate_device_entries(CriuKfd *e, int num_of_devices)
 {
-	e->devinfo_entries = xmalloc(sizeof(DevinfoEntry*) * num_of_devices);
-	if (!e->devinfo_entries) {
-		pr_err("Failed to allocate devinfo_entries\n");
+	e->device_entries = xmalloc(sizeof(DeviceEntry*) * num_of_devices);
+	if (!e->device_entries) {
+		pr_err("Failed to allocate device_entries\n");
 		return -1;
```

Comment:
-ENOMEM

Reply:
ACK for -ENOMEM
pr_perror should only be used then errno is set